### PR TITLE
chore: add missing license headers to files

### LINKED ISSUE #81: Add license headers to files
I ran the code in this repo over a script that checks for repo setup and it found the following issues:

    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.Samples/SampleModel/SampleDataModel.sql: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/TableSplittingTests/Migrations/20210224080116_Initial.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/MigrationTests/Migrations/TestMigrationDbContextModelSnapshot.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/MigrationTests/Migrations/20210324085522_Initial.Designer.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/TableSplittingTests/Migrations/TestTableSplittingDbContextModelSnapshot.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/MigrationTests/Migrations/20210324085522_Initial.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.Tests/MigrationTests/Migrations/20210309110233_Initial.Designer.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.Tests/MigrationTests/Migrations/20210309110233_Initial.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/Model/Venues.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/Model/Albums.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/Model/Concerts.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/Model/Performances.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/Model/Singers.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/TableSplittingTests/Migrations/20210224080116_Initial.Designer.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/Model/SpannerSampleDbContext.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/Model/TableWithAllColumnTypes.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SampleDataModel - Emulator.sql: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.Tests/MigrationTests/SampleMigrationDataModel.sql: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.Tests/MigrationTests/Migrations/MockMigrationSampleDbContextModelSnapshot.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner/Properties/AssemblyInfo.cs: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SampleDataModel.sql: Missing a required license header (or its header was not recognized)
    - ❌ /Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/Model/Tracks.cs: Missing a required license header (or its header was not recognized)

All files need to have a license header so if you could please fix these, we'd really appreciate it.

### PR DESCRIPTION
**Pull request by laljikanjareeya**
Fixes #81
Need to skip license header check for `.sql` files.

### CODE REVIEW

**Review by olavloite — APPROVED** [review]

**Review by skuruppu — APPROVED** [review]

**Comment by skuruppu** [comment]
@laljikanjareeya I see that it's awkward to add headers to sql files. It seems that we won't be able to move this repo to googleapis without it (we may not get the approvals). I'll give it a try but we may have to find a way to add the headers.

**Comment by laljikanjareeya** [comment]
> @laljikanjareeya I see that it's awkward to add headers to sql files. It seems that we won't be able to move this repo to googleapis without it (we may not get the approvals). We had to do the same [here](https://github.com/cloudspannerecosystem/spanner-benchmarks-tutorial/blob/main/data/latency_benchmarks/workloada.yaml) for yaml configs.
> 
> I'll try to get it approved but we may have to find a way to add the headers.

I will take a look.

**Comment by laljikanjareeya** [comment]
@skuruppu  @olavloite license headers added in `.sql` file and remove it while in test. PTAL.

File: Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerSampleFixture.cs
Comment (olavloite) [inline Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerSampleFixture.cs:131]: Is it at all necessary to remove it from the script? The copy right header is also a valid comment in Cloud Spanner SQL syntax, so I would expect Cloud Spanner to just ignore the comment and execute the actual statement that follows it.

Comment (laljikanjareeya) [reply Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerSampleFixture.cs:131]: I was also thinking same but actually it's throwing Invalid Argument Exception:
`
Status(StatusCode="InvalidArgument", Detail="Error parsing Spanner DDL statement: /* Copyright 2021 Google LLC\r\n* \r\n* Licensed under the Apache License, Version 2.0 (the \"License\")\r\n* you may not use this file except in compliance with the License.\r\n* You may obtain a copy of the License at\r\n* \r\n*     https://www.apache.org/licenses/LICENSE-2.0\r\n* \r\n* Unless required by applicable law or agreed to in writing, software\r\n* distributed under the License is distributed on an \"AS IS\" BASIS,\r\n* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\r\n* See the License for the specific language governing permissions and\r\n* limitations under the License.\r\n*/\r\n\r\nCREATE TABLE Singers (\r\n  SingerId  INT64 NOT NULL,\r\n  FirstName STRING(200),\r\n  LastName  STRING(200) NOT NULL,\r\n  FullName  STRING(400) NOT NULL AS (COALESCE(FirstName || \' \', \'\') || LastName) STORED,\r\n  BirthDate DATE,\r\n  Picture   BYTES(MAX),\r\n) PRIMARY KEY (SingerId) : Syntax error on line 1, column 1: Encountered \'/\' while pa...", DebugException="Grpc.Core.Internal.CoreErrorDetailException: {"created":"@1620917204.223000000","description":"Error received from peer ipv4:142.250.183.234:443","file":"T:\src\github\grpc\workspace_csharp_ext_windows_x64\src\core\lib\surface\call.cc","file_line":1062,"grpc_message":"Error parsing Spanner DDL statement: /* Copyright 2021 Google LLC\r\n* \r\n* Licensed under the Apache License, Version 2.0 (the \"License\")\r\n* you may not use this file except in compliance with the License.\r\n* You may obtain a copy of the License at\r\n* \r\n*     https://www.apache.org/licenses/LICENSE-2.0\r\n* \r\n* Unless required by applicable law or agreed to in writing, software\r\n* distributed under the License is distributed on an \"AS IS\" BASIS,\r\n* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\r\n* See the License for the specific language governing permissions and\r\n* limitations under the License.\r\n*/\r\n\r\nCREATE TABLE Singers (\r\n  SingerId  INT64 NOT NULL,\r\n  FirstName STRING(200),\r\n  LastName  STRING(200) NOT NULL,\r\n  FullName  STRING(400) NOT NULL AS (COALESCE(FirstName || \' \', \'\') || LastName) STORED,\r\n  BirthDate DATE,\r\n  Picture   BYTES(MAX),\r\n) PRIMARY KEY (SingerId) : Syntax error on line 1, column 1: Encountered \'/\' while pa...","grpc_status":3}")
`

Comment (olavloite) [reply Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerSampleFixture.cs:131]: Strange, it seems that Cloud Spanner only accepts comments in SQL queries and DML statements, but not in DDL statements.

**Review by olavloite — APPROVED** [review]

File: Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerSampleFixture.cs
Comment (skuruppu) [reply Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerSampleFixture.cs:131]: Ah, that's a shame. Ok we can just remove those and I'll explain to the approvers that it's not possible to add headers to SQL files. I also noticed that the sql files in the JDBC repo don't have a header either.

Comment (skuruppu) [reply Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerSampleFixture.cs:131]: I take my last comment back. In the JDBC repo we have https://github.com/googleapis/java-spanner-jdbc/blob/master/src/test/resources/com/google/cloud/spanner/jdbc/ITSqlScriptTest_CreateTables.sql, which has DDL statements so Spanner should be able to accept this, no?

File: Google.Cloud.EntityFrameworkCore.Spanner.Tests/MigrationTests/GenerateCreateScriptTest.cs
Comment (skuruppu) [inline Google.Cloud.EntityFrameworkCore.Spanner.Tests/MigrationTests/GenerateCreateScriptTest.cs:44]: Is this now inlined to avoid the license check or is it for some other reason?

Comment (laljikanjareeya) [reply Google.Cloud.EntityFrameworkCore.Spanner.Tests/MigrationTests/GenerateCreateScriptTest.cs:44]: Yes this is to avoid the license header check.

**Comment by skuruppu** [comment]
I will merge this in. Thanks @laljikanjareeya.

File: Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerSampleFixture.cs
Comment (olavloite) [reply Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerSampleFixture.cs:131]: I had forgotten as well, but the JDBC driver (and the Java client library) strip the copyright header before executing any statements from these SQL scripts: https://github.com/googleapis/java-spanner/blob/eb25eae4a2f83fd8c35907a87785baf042889233/google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/AbstractSqlScriptVerifier.java#L169

**Pull request merged**
